src/components/map-context.js

```diff
@@ -0,0 +1,17 @@
+import {createContext} from 'react';
+
+export default createContext({
```

Comment:
I am curious why these but not others needed to be in `MapContext`?

Reply:
Examples for "others"? These are information our existing controls need to render properly. We do add new props to the context if they are identified as useful.